opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/AbstractJettyTest.java

```diff
@@ -182,4 +184,15 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         @Override
         public void destroy() {}
     }
+
+    public static class CurrentSpanServlet extends HttpServlet {
```

Comment:
Could you please move this below `LocalSpanServlet` and the same with `        servletContext.addServlet(CurrentSpanServlet.class, "/currentSpan");
`